src/routes_initializer.coffee

```diff
@@ -17,12 +17,14 @@ class RoutesInitializer
 
   constructor: (@routesFile, @reverseRoutes, @addRoute) ->
     @_routes = []
+    @_scope = ''
     require(@routesFile) @getRouteDirectives()
 
 
   # Returns a route directive for a specific http method to be called in a routes file
   getHttpDirective: (httpMethod) ->
     (url, {to, as}) =>
+      url = "#{@_scope}#{url}"
```

Comment:
@charlierudolph So the standard routes are:
`GET '/users', to: 'users#index'`
It seems that if we scope things you should not expect it to be different just because it is scoped.
So I would expect:
```
scope '/api', ->
  GET '/users', to: 'users#index'
```
Not
```
scope '/api', ->
  GET 'users', to: 'users#index'
```

That being said, I do think we should be lenient and allow it to be left off. In theory we should probably allow it to be left off the scope name too.

So I like Alex's idea with a minor modification:
`path.posix.normalize "/#{@_scope}/#{url}"`